app/views/users/_toolbar.html

```diff
@@ -0,0 +1,32 @@
+
+<div class="toolbar-container">
+  <div id="toolbar">
+    <div class="title-container">
+      <h2><%= link_to l(:label_user_plural), :controller => '/users', :action => 'index' %> &#187; <%= new_user ? l("label_user_new") : @user.login %></h2>
```

Comment:
It is wrong to always display `@user.login`. The correctly formatted string should be available under `@user.name` (taking instance settings into account).